client/src/components/formHelpers/FormValidators.js

```diff
@@ -0,0 +1,29 @@
+// Matches editor links for: Repl.it, Glitch, CodeSandbox
+const editorRegex = /repl\.it\/@|edit\/#!|codesandbox\.io\/s\//;
+const localhostRegex = /localhost/;
```

Comment:
I think this would catch /localhost/ if it is present anywhere in the url. It would be better if we could make it more specific.